src/SourceGenerators/Uno.UI.SourceGenerators.net6.Tests/Utilities/ISourceGeneratorExtensions.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+
+namespace Uno.UI.SourceGenerators.Tests;
```

Comment:
Also, let's add some new documentation about those tests here: https://github.com/unoplatform/uno/blob/master/doc/articles/contributing/guidelines/creating-tests.md#types-of-tests